api/conversion.go

```diff
@@ -92,6 +80,14 @@ func (a Endpoint) toDb() db.Endpoint {
 	}
 }
 
+func jwkToMap(jwk []JWK) []interface{} {
```

Comment:
nuts-crypto has this in /pkg as utility function

Reply:
I'm not sure how these 2 functions compare, since their implementation is quite different. This function looks new, but I just moved it over from Organization.go. I'd prefer to fix this in a later PR.